apps/baseline/models.py

```diff
@@ -279,19 +290,38 @@ class WorksheetName(models.TextChoices):
         verbose_name=_("Livelihood Zone Baseline"),
     )
     worksheet_name = models.CharField(max_length=20, choices=WorksheetName.choices, verbose_name=_("Worksheet name"))
-    cell_range = models.CharField(max_length=10, verbose_name=_("Cell range"))
-    previous_value = models.CharField(max_length=255, verbose_name=_("Previous value before correction"))
-    value = models.CharField(max_length=255, verbose_name=_("Corrected value"))
+    cell_range = models.CharField(max_length=20, verbose_name=_("Cell range"))
+    previous_value = models.JSONField(verbose_name=_("Previous value before correction"))
+    value = models.JSONField(verbose_name=_("Corrected value"))
     correction_date = models.DateTimeField(auto_now_add=True, verbose_name=_("Correction date"))
     author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, verbose_name=_("Correction author"))
     comment = models.TextField(
         max_length=255,
         verbose_name=_("Correction comments"),
         help_text=_("Required comment about the correction suggested"),
     )
+    objects = LivelihoodZoneBaselineCorrectionManager()
+
+    class Meta:
+        verbose_name = _("Livelihood Zone Baseline Correction")
+        verbose_name_plural = _("Livelihood Zone Baseline Corrections")
+        constraints = [
+            models.UniqueConstraint(
+                fields=("livelihood_zone_baseline", "worksheet_name", "cell_range"),
+                name="livelihood_zone_baseline_corrections_uniq",
```

Comment:
The "correct" name is made up of the database table name and the fields in the constraint, so `baseline_livelihoodzonebaselinecorrection_livelihood_zone_baseline_worksheet_name_cell_range_uniq`.

Try that, if that is too long, fall back to just the table name: `baseline_livelihoodzonebaselinecorrection_uniq`

Reply:
The max character for identifier is 63, it didn't complain when I give it `baseline_livelihoodzonebaselinecorrection_livelihood_zone_baseline_worksheet_name_cell_range_uniq` but truncated the last bit as its len is 97, so updated to the fallback `baseline_livelihoodzonebaselinecorrection_uniq`